UseSecurityFeatures.md

```diff
@@ -94,11 +107,19 @@ Secure boot enabled, so bootloader not flashed automatically.
 ==============================================================================
 ```
 2. Flash the bootloader by copying and pasting the command under "Secure boot
-enabled, so bootloader not flashed automatically," (the second block of text)
+enabled, so bootloader not flashed automatically," (the second block of text),
 replacing:
-**PORT** with the serial port to which the ESP32-C3 is connected. (Do not include
-the opening and closing braces around PORT in the command)
-**BAUD** with 460800.
+    - **(PORT)** with the serial port to which the ESP32-C3 is connected.
+     (Do not includethe opening and closing braces around PORT in the command)
+    - **(BAUD)** with 460800.
+
+**NOTE**: If bootloader image is too big that exceeds bootloader size limit,
```

Comment:
I think we should just change the default bootloader verbosity to ERROR or WARN instead of doing this. I don't believe the build command will tell you if the bootloader is too big or small. Where this just leads to the following flash failing, and then needing to re-do all this.

Reply:
We could do that.  It is just it is not happening with c3 board.